Example/Emission.xcodeproj/project.pbxproj

```diff
@@ -222,12 +225,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 510DCB3F1CCA69ED0075E8CB /* Build configuration list for PBXNativeTarget "EmissionTests" */;
 			buildPhases = (
+				20B6FFACE9798C3F6AEDEF21 /* [CP] Check Pods Manifest.lock */,
```

Comment:
Where are the `Podfile.lock` changes?

I’m asking, because those emojis that get added to the `project.pbxproj` file were only in 1.0.0, iirc. From [the CP CHANGELOG](https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md#101-2016-06-02):

> Removed emojis in Build Phases names